https://issues.redhat.com/browse/ACM-20524 for 2.13 (#7940)
open-cluster-management/rhacm-docs: clusters/install_upgrade/install_connected.adoc

* Your cluster does _not_ have a `ManagedCluster` resource configured in an external cluster.

> * You must have a supported version of {ocp-short} deployed in your environment, and you must log in from the {ocp-short} command-line interface. See link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installation overview] in the {ocp-short} documentation.

Reviewer: So users don't feel obligated to go to this link, then leave your doc for no reason, perhaps give a reason why they would go to the link, and if it's inapplicable to them, they can move on to the next prereq? 

Putting a link in a prereq is kind of like telling the users to go to the link. What do you think of something like this? 

```suggestion
* You must have a supported version of {ocp-short} deployed in your environment, and you must log in from the {ocp-short} command-line interface. To ensure you have a supported versino, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installation overview] in the {ocp-short} documentation.
```

Author: i can understand how that can help. However, it is safe to assume that a user is obligated to click the link if they are unaware of the installation process for OCP. What do you think about my suggestion? This seems wordy to me. i prefer to simply write "See the Installation overview...". 
```suggestion
* You must have a supported version of {ocp-short} deployed in your environment, and you must log in from the {ocp-short} command-line interface. See the supported {ocp-short} version documentation, link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installation overview].
```

> * You must have a supported version of {ocp-short} deployed in your environment, and you must log in from the {ocp-short} command-line interface. See link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installation overview] in the {ocp-short} documentation.

Reviewer: or:

```suggestion
* You must have a supported version of {ocp-short} deployed in your environment, and you must log in from the {ocp-short} command-line interface. For help with logging into the {ocp-short} command-line interface, see the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installation overview] in the {ocp-short} documentation.
```

Reviewer: I would go with something more like my first suggestion as the doc we're linking to (installation) seems more like help with installing a supported version.

Author: I like this suggestion more actually.

> * You must have a supported version of {ocp-short} deployed in your environment, and you must log in from the {ocp-short} command-line interface. For help with logging into the {ocp-short} command-line interface, see the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installation overview] in the {ocp-short} documentation.

Author: ```suggestion
* You must have a supported version of {ocp-short} deployed in your environment, and you must log in from the {ocp-short} command-line interface. For help with installing {ocp-short}, see the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/installation_overview/index[Installation overview] in the {ocp-short} documentation.
```

> * Your {ocp-short} command-line interface must be configured to run `oc` commands. See link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.16/html/cli_tools/openshift-cli-oc#cli-getting-started[Getting started with the CLI] for information.

Reviewer: I know it's a bit wordier, but the link is only helpful to users if they need it, but putting it in a prereq kind of prompts the users to click the link, even if they don't need it. Maybe a brief intro, like this? : 

```suggestion
* Your {ocp-short} command-line interface must be configured to run `oc` commands. For help with the command-line interface, see link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.16/html/cli_tools/openshift-cli-oc#cli-getting-started[Getting started with the CLI] for information.
```

[#installing-from-the-operatorhub-mce]
== Installing from the OperatorHub web console interface

> *Best practice:* From the _Administrator_ view in your {ocp-short} navigation, install the OperatorHub web console interface that is provided with {ocp-short}.

Reviewer: Honestly, I don't think we need the `Best practice` Our docs should predominately only include the best, most effective approach. Also, now we don't have a short description, we have a best practice note instead, lol. 

How do you think this reads? 

```suggestion
From the _Administrator_ view in your {ocp-short} navigation, install the OperatorHub web console interface that is provided with {ocp-short}. Complete the following steps: 
```

Author: That's a great point.
